pg_temp/pg_temp.py

```diff
@@ -358,6 +359,8 @@ def cleanup(self):
                 stdout=subprocess.DEVNULL,
             )
         elif self.pg_process:
+            pg_stop_cmd = ['pg_ctl', 'stop', '-D', self.pg_data_dir, '-m' 'fast']
```

Comment:
We need to be able to override pg_ctl in the same way that we do postgres or initdb, in the case the user is using an alternative postgres install that is not in the PATH. This needs a default, and an argument (plus said default) in the constructor. See for example `DEFAULT_POSTGRES` or `DEFAULT_INITDB`